src/logic/operator.js

```diff
@@ -0,0 +1,22 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    return one.div(two).toString();
+  }
```

Comment:
- [ ] I am getting an error when I want to divide a number by `zero`
![e](https://user-images.githubusercontent.com/44745855/129456926-32cedccc-ff21-4fcf-bb69-b9d2842bdbce.png)

Normally it has to display an `error` to the screen,please fix that

Reply:
Resolved

Reply:
Hi